engine.js

```diff
@@ -61,7 +61,7 @@ module.exports = function (options) {
         }, {
           type: 'input',
           name: 'subject',
-          message: 'Write a short, imperative tense description of the change:\n'
+          message: 'Write a short, imperative terse description of the change:\n'
```

Comment:
Imperative tense here refers to the [Imperative Mood](https://en.wikipedia.org/wiki/Imperative_mood), it was not meant to mean a terse (short) description.

Reply:
Thank you @LinusU -- that makes so much sense now. My apologies.